RELEASE_NOTE.txt

```diff
@@ -1,13 +1,53 @@
 Release Notes:
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Google Search Appliance

```

Comment:
Sorry, I misremembered the name fix. It's just dropping "Google", so "SAML Bridge for Windows".